test_project/rename_forward/urls.py

```diff
@@ -8,6 +8,7 @@
 class LinkedDataView(autocomplete.Select2QuerySetView):
     def get_queryset(self):
         qs = super(LinkedDataView, self).get_queryset()
+        print(self.forwarded.get('checkbox', None))
```

Comment:
Of course, that's purely for testing purposes. I didn't have idea for a meaningful test, but had to test it somehow.